pkgs/by-name/ap/appmenu-glib-translator/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  fetchFromGitLab,
+  glib,
+  gtk3,
+  meson,
+  ninja,
+  pkg-config,
+  gobject-introspection,
+  vala,
+  stdenv,
+  wrapGAppsHook3,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "appmenu-glib-translator";
+  version = "25.04";
+
+  src = fetchFromGitLab {
+    owner = "vala-panel-project";
+    repo = "vala-panel-appmenu";
+    tag = finalAttrs.version;
+    fetchSubmodules = true;
+    hash = "sha256-v5J3nwViNiSKRPdJr+lhNUdKaPG82fShPDlnmix5tlY=";
+  };
+
+  sourceRoot = "source/subprojects/appmenu-glib-translator";
+
+  nativeBuildInputs = [
+    meson
+    ninja
+
+    pkg-config
+    wrapGAppsHook3
+    vala
+  ];
+
+  buildInputs = [
+    glib
+    gobject-introspection
+    gtk3
```

Comment:
There is only gdk-pixbuf dependency, no GTK:

https://gitlab.com/vala-panel-project/vala-panel-appmenu/-/blob/f05d28d805a22a7564895aa178772361c44b6b7a/subprojects/appmenu-glib-translator/meson.build#L15